ml_datasets/loaders/cifar.py

```diff
@@ -0,0 +1,81 @@
+import pickle
+import tarfile
+import random
+import numpy
+
+from ..util import get_file, unzip, to_categorical
+
+CIFAR10_URL = "https://www.cs.toronto.edu/~kriz/cifar-10-python.tar.gz"
+CIFAR100_URL = "https://www.cs.toronto.edu/~kriz/cifar-100-python.tar.gz"
+
+
+def cifar(variant='10', channels_last=False, shuffle=True):
+    if variant == '10':
+        data = load_cifar10()
+    elif variant == '100':
+        data = load_cifar100()
```

Comment:
In these cases I think it's better to be explicit and robust against possible future changes of default values (even if that's not very likely)
```suggestion
        data = load_cifar100(coarse=False)
```

Reply:
thanks, good point! I haven't thought about this before.